.github/workflows/on-pr-debug.yml

```diff
@@ -98,4 +106,4 @@ jobs:
       - name: Test
         run: |
           cd build
-          ./bin/Debug/testsuite
+          ctest -L 'test' --timeout 60 --output-on-failure
```

Comment:
cc @mtmk `ctest` seems to work fine?

Reply:
doesn't run any tests
https://github.com/nats-io/nats.c/actions/runs/10116438126/job/27979286329?pr=774#step:5:10
```
Test project D:/a/nats.c/nats.c/build
No tests were found!!!
```

Reply:
@mtmk I think I cleaned it up well, borrowing from your PR. Not sure why many tests are flaky, for now retry=3. Please review when you have a chance.